pkg/virtctl/pause/pause.go

```diff
@@ -87,22 +87,15 @@ func executePauseCMD(client kubecli.KubevirtClient, namespace, resourceType, res
 		if err != nil {
 			return fmt.Errorf("Error getting VirtualMachine %s: %v", resourceName, err)
 		}
-		vmiName := vm.Name
-		err = client.VirtualMachineInstance(namespace).Pause(context.Background(), vmiName, &kubevirtV1.PauseOptions{DryRun: dryRunOption})
+
+		err = client.VirtualMachineInstance(namespace).Pause(context.Background(), vm.Name, &kubevirtV1.PauseOptions{DryRun: dryRunOption})
+		if errors.IsNotFound(err) {
+			return handleNotFoundError(vm, err)
+		}
 		if err != nil {
-			if errors.IsNotFound(err) {
-				runningStrategy, err := vm.RunStrategy()
-				if err != nil {
-					return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
-				}
-				if runningStrategy == kubevirtV1.RunStrategyHalted {
-					return fmt.Errorf("Error pausing VirtualMachineInstance %s. VirtualMachine %s is not set to run", vmiName, vm.Name)
-				}
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s, it was not found", vmiName)
-			}
-			return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+			return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vm.Name, err)
 		}
-		fmt.Printf("VMI %s was scheduled to pause\n", vmiName)
+		fmt.Printf("VMI %s was scheduled to pause\n", vm.Name)
```

Comment:
One more thing: You should be able to move this print out of both switch cases, so it is not duplicated.